streaming-pacman/pacman/game/js/pacman.js

```diff
@@ -43,7 +43,7 @@ function resetPacman() {
 	PACMAN_DEAD = false;
 	PACMAN_SUPER = false;
 }
-function getPacmanCanevasContext() { 
+function getPacmanCanevasContext() { //TODO: is this a typo? I assume so
```

Comment:
yes, but it is wrongly written also at usage time. If you want you can fix it in both places :)